src/scene/shader-lib/chunks/chunks.js

```diff
@@ -256,6 +265,15 @@ const shaderChunks = {
     gles3PS,
     gles3VS,
     glossPS,
+    gsplatPS,
+    gsplatVS,
+    gsplatCommonVS,
+    gsplatCompressedDataVS,
+    gsplatCompressedSHVS,
```

Comment:
I'm surprised Will has not mentioned yet those need to be in alphabetical order.

Reply:
...

Reply:
They need to be in alphabetical order.